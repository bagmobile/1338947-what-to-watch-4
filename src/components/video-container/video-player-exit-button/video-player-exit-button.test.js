import React from "react";
import renderer from "react-test-renderer";
import VideoPlayerExitButton from "./video-player-exit-button";

describe(`VideoPlayerExitButton`, () => {

  it(`Render`, () => {
    const tree = renderer.create(
        <VideoPlayerExitButton
          onClick={()=>{}}
        />
    );
    expect(tree).toMatchSnapshot();
  });

});
