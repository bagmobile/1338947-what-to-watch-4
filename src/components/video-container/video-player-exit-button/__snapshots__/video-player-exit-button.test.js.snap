// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerExitButton Render 1`] = `
<button
  className="player__exit"
  onClick={[Function]}
  type="button"
>
  Exit
</button>
`;
